content/id/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html

```diff
@@ -0,0 +1,109 @@
+---
+title: Menggunakan kubectl untuk membuat Deployment
+weight: 10
+---
+
+<!DOCTYPE html>
+
+<html lang="id">
+
+<body>
+
+    <link href="/docs/tutorials/kubernetes-basics/public/css/styles.css" rel="stylesheet">
+
+<div class="layout" id="top">
+
+    <main class="content">
+
+        <div class="row">
+
+         <div class="col-md-8">
+          <h3>Tujuan</h3>
+                <ul>
+                    <li>Belajar tentang aplikasi Deployment.</li>
+                    <li><i>Deploy</i> aplikasi pertama kamu di Kubernetes dengan kubectl.</li>
+                </ul>
+            </div>
+
+            <div class="col-md-8">
+                <h3><i>Deployment</i> Kubernetes</h3>
+                <p>
+                Ketika klaster Kubernetes kamu sudah berjalan, kamu dapat men-<i>deploy</i> aplikasi terkontainerisasi di atasnya.
+                Untuk melakukannya, kamu perlu membuat konfigurasi <b>Deployment</b>. Deployemnt menginstruksikan Kubernetes
+                tentang bagaimana cara membuat dan memperbaharui <i>instance</i> aplikasi kamu. Ketika kamu selesai membuat Deployment, Kubernetes master
+                melakukan penjadwalan <i>instance</i> aplikasi yang terlibat dalam Deployment untuk berjalan di Node individu pada klaster.
+                </p>
+
+                <p>Setelah <i>instance</i> aplikasi telah dibuat, Kubernetes Deployment Controller akan melakukan monitoring <i>instance</i> tersebut secara kontinu. Jika <i>instance</i> pada hos Node <i>down</i> atau terhapus, Deployment Controller akan mengganti <i>instance</i> tersebut dengan <i>instance</i> Node lain dalam klaster. <b>Ia menyediakan mekanisme penyembuhan diri (<i>self-healing</i>) untuk mengatasi kegagalan mesin atau pemeliharaan (<i>maintenance</i>)</b></p>
+
+                <p>Pada masa pra-orkestrasi, instalasi <i>script</i> sering digunakan untuk memulai aplikasi, namun cara ini tidak memberikan mekanisme pemulihan ketika terjadi kegagalan mesin. Dengan kemampuan membuat <i>instance</i> aplikasi dan menjaganya tetap berjalan pada Node, Kubernetes Deployment menyediakan manajemen aplikasi dengan pendekatan fundamental yang berbeda. </p>
+
+            </div>
+
+            <div class="col-md-4">
+                <div class="content__box content__box_lined">
+                    <h3>Summary:</h3>
+                    <ul>
+                        <li>Deployment</li>
+                        <li>Kubectl</li>
+                    </ul>
+                </div>
+                <div class="content__box content__box_fill">
+                    <p><i>
+                        Sebuah Deployment bertanggung jawab untuk membuat dan memperbaharui instance dari aplikasi kamu
```

Comment:
https://kbbi.kemdikbud.go.id/entri/memperbarui
```suggestion
                        Sebuah Deployment bertanggung jawab untuk membuat dan memperbarui instance dari aplikasi kamu
```